streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -194,8 +219,8 @@ public void testPutAndFetch() throws IOException {
 
     @SuppressWarnings("unchecked")
     @Test
-    public void shouldGetAll() throws IOException {
-        windowStore = createWindowStore(context, false, true);
+    public void shouldGetAll() {
+        windowStore = createWindowStore(context);
```

Comment:
again, and some more time below (wondering why tests pass? Or are do I miss something?)

Reply:
Only 2 out of these tests requires retain duplicates to true, while before this patch all tests requires it to be true, which is an overkill and may hide issue though the test still passes..

Reply:
Ack. Was not sure if it's an intentional change. Thanks for clarifying that it's an actual fix.